[2025-01-31T21:40:07.969+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T21:40:08.020+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_processing.nds-load scheduled__2025-01-31T21:38:00+00:00 [queued]>
[2025-01-31T21:40:08.041+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_processing.nds-load scheduled__2025-01-31T21:38:00+00:00 [queued]>
[2025-01-31T21:40:08.041+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-31T21:40:08.076+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): nds-load> on 2025-01-31 21:38:00+00:00
[2025-01-31T21:40:08.090+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3055) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T21:40:08.094+0000] {standard_task_runner.py:72} INFO - Started process 3060 to run task
[2025-01-31T21:40:08.096+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_processing', 'nds-load', 'scheduled__2025-01-31T21:38:00+00:00', '--job-id', '664', '--raw', '--subdir', 'DAGS_FOLDER/DAG_etl.py', '--cfg-path', '/tmp/tmprswlrph8']
[2025-01-31T21:40:08.104+0000] {standard_task_runner.py:105} INFO - Job 664: Subtask nds-load
[2025-01-31T21:40:08.246+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_processing.nds-load scheduled__2025-01-31T21:38:00+00:00 [running]> on host a664bcf281bf
[2025-01-31T21:40:08.980+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_processing' AIRFLOW_CTX_TASK_ID='nds-load' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T21:38:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-31T21:38:00+00:00'
[2025-01-31T21:40:08.983+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T21:40:09.056+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-31T21:40:09.136+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-31T21:40:09.281+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-31T21:40:09.421+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-31T21:40:09.583+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-31T21:40:09.722+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-31T21:40:09.876+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-31T21:40:10.018+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-31T21:40:10.214+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-31T21:40:10.343+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-31T21:40:10.392+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T21:40:10.394+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_processing, task_id=nds-load, run_id=scheduled__2025-01-31T21:38:00+00:00, execution_date=20250131T213800, start_date=20250131T214008, end_date=20250131T214010
[2025-01-31T21:40:10.470+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T21:40:10.569+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-31T21:40:10.573+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
